bigtable/docs/snippets.py

```diff
@@ -38,8 +38,8 @@
 from google.cloud.bigtable import enums
 
 
-INSTANCE_ID = "snippet-" + unique_resource_id("-")
-CLUSTER_ID = "clus-1-" + unique_resource_id("-")
+INSTANCE_ID = "snippet" + unique_resource_id("-")
```

Comment:
This looks very similar to another BT PR I reviewed, perhaps this has a diffbase? https://github.com/googleapis/google-cloud-python/pull/7033/files